src/live.asp.net/Controllers/HomeController.cs

```diff
@@ -37,6 +37,14 @@ public HomeController(IShowsService showsService, ILiveShowDetailsService liveSh
             });
         }
 
+        [HttpGet("/feed")]
+        [Produces("application/rss+xml")]
+        public async Task<string> GetRssFeed()
+        {
+            var feedService = new FeedService(_showsService, false);
```

Comment:
Why you set the `disableCache` false all the time?!! if that's your intend I don't think its needed in the constructor, you can passing it directly in `GetRecordedShowsAsync`